lib/pyqmstr/qmstr/module/utils.py

```diff
@@ -0,0 +1,87 @@
+import os
+from tempfile import mkdtemp
+import hashlib
+import json
+import logging
+from qmstr.service.datamodel_pb2 import FileNode, InfoNode, PackageNode
+
+
+def _filter_out_hidden_files(path_list):
+    """
+    Returns the path list ignoring all paths and files starting with a dot.
+    """
+    # FIXME: is it needed?
```

Comment:
Should we remove this comment?
I think it's important to keep this filter to avoid future problems.